docker-compose.yaml

```diff
@@ -0,0 +1,27 @@
+version: '3'
+
+services:
+    composer:
+        build:
+            context: .
+            dockerfile: Dockerfile.composer
+            args:
+                composer_dev_arg: ${COMPOSER_DEV_ARG}
+        image: libero/browser_composer:${IMAGE_TAG}
+        command: /bin/true
+    fpm:
+        build:
+            context: .
+            args:
+                image_tag: ${IMAGE_TAG}
+        image: libero/browser:${IMAGE_TAG}
```

Comment:
Should we parameterise the image name too?

Reply:
For creating `libero/browser_unstable:12345` etc? Can do if needed (maybe `${IMAGE_SUFFIX}`?), can this be done by whatever's going to create that image?

Reply:
The build can pass in the necessary `IMAGE_SUFFIX`, but it needs a parameter to be added here. Not a blocker.